test/e2e/storage/in_tree_volumes.go

```diff
@@ -39,7 +39,6 @@ var testDrivers = []func() storageframework.TestDriver{
 	drivers.InitVSphereDriver,
 	drivers.InitAzureDiskDriver,
 	drivers.InitAzureFileDriver,
-	drivers.InitAwsDriver,
```

Comment:
Do you need add something like this for AWS?
https://github.com/kubernetes/kubernetes/pull/109541/files#diff-5f786e5e2c507b54cd793523941bc370fb2c3a8894efbc1b79c97a512415e057R56

Reply:
Done! 

Using the command line option `"enabled-volume-drivers"` instead of the environment variable:
https://github.com/kubernetes/kubernetes/blob/b3926d137cd2964cd3a04088ded30845910547b1/test/e2e/framework/test_context.go#L393